src/Avalonia.Controls/Platform/Screen.cs

```diff
@@ -1,21 +1,42 @@
 ﻿namespace Avalonia.Platform
 {
+    /// <summary>
+    /// Represents a single display screen.
+    /// </summary>
     public class Screen
     {
+        /// <summary>
+        /// Gets the pixel density of the screen.
+        /// This is a scaling factor so multiply by 100 to get a percentage.
```

Comment:
I think this is the scaling factor set by the OS, but "pixel density of the screen" makes me think physical DPI. Maybe should be renamed to `Scaling`?

Also:

"This is a scaling factor so multiply by 100 to get a percentage." might be better in `<remarks>`?

Reply:
This property is the one I was most unsure of. I wrote the first sentence based on the property name -- then I wrote the second sentence after seeing how the property was actually used in code. So your clarifying details are helpful here. I agree the property was misnamed as well which doesn't help the confusion.

WinUI has the DisplayInformation class which is most similar here. That class has the [`ResolutionScale`](https://learn.microsoft.com/en-us/uwp/api/windows.graphics.display.displayinformation.resolutionscale?view=winrt-22621) property. Since there is only one scale though, I don't think qualifying it for "resolution" is really necessary. Therefore, for now I'm just naming this new property `Scale` in Avalonia.

Reply:
Yeah I took a look around and we seem to [use](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Base/Rendering/IRenderRoot.cs#L26) [`Scaling`](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Base/Layout/ILayoutRoot.cs#L19) in general for this concept so I'd go with that.